pages/services/data-science-engine/1.0.0/examples/index.md

```diff
@@ -184,8 +184,32 @@ sum
 # Advanced
 
 ## Launching a Spark job
-Open a `Python Notebook` and put the following code in a code cell.
+### Using Terminal
```

Comment:
All of this is good but can you edit the sections for R, Groovy and Clojure? All of them say `put the following sections in different code cells` but there's only one cell's worth of code. Can you change the text to something like:
```
Open a `Groovy Notebook` and run the following inside a code cell:
```